test/index.html

```diff
@@ -0,0 +1,16 @@
+<!doctype html>
+<html>
+	<head>
+		<meta charset="utf-8">
+		<script src="../node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+		<script src="../node_modules/mocha/mocha.js"></script>
+	    <script src="../node_modules/wct-mocha/wct-mocha.js"></script>
+	</head>
+	<body>
+		<script>
+			WCT.loadSuites([
+				'./button/button-subtle.html'
```

Comment:
Talked to Dave B about this and confirmed that the `?wc-shadydom=true` parameter is no longer working.  If we care, we can set it in JS, but I argue we no longer care... now that each browser will only ever be in its default state, we can just test each browser in that state.

Reply:
Agreed.